docroot/themes/contrib/civictheme/civictheme_library/components/00-base/collapsible/collapsible.js

```diff
@@ -53,7 +53,9 @@ function CivicThemeCollapsible(el) {
   this.trigger.addEventListener('focusout', this.focusoutEvent.bind(this));
   this.panel.addEventListener('click', (e) => e.stopPropagation());
   this.panel.addEventListener('focusout', this.focusoutEvent.bind(this));
-  this.panel.addEventListener('focusin', this.focusinEvent.bind(this));
```

Comment:
removed `focusin` listener and everything related to handling and calculating of "focusability" of the element